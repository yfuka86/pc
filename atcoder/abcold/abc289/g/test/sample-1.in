5 4
100 200 300 400 500
120 370 470 80
