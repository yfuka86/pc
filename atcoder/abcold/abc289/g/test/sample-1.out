1280 2350 2850 1140
